corehq/apps/reports/templates/reports/project_health/project_health_email.html

```diff
@@ -0,0 +1,85 @@
+{% extends "reports/async/default.html" %}
```

Comment:
what does this get you? it looks like it has a lot of js/css stuff in it that likely doesn't render properly in email.